realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/utils/HttpUtils.java

```diff
@@ -56,62 +61,27 @@ public static void startSyncServer() throws Exception {
         for (int i = 0; i < responseHeaders.size(); i++) {
             Log.d(TAG, responseHeaders.name(i) + ": " + responseHeaders.value(i));
         }
-
         Log.d(TAG, response.body().string());
-
-        // FIXME: Server ready checking should be done in the control server side!
-        if (!waitAuthServerReady()) {
-            stopSyncServer();
-            throw new RuntimeException("Auth server cannot be started.");
-        }
     }
 
-    // Checking the server
-    private static boolean waitAuthServerReady() throws InterruptedException {
-        int retryTimes = 20;
-
-        // Dummy invalid request, which will trigger a 400 (BAD REQUEST), but indicate the auth
-        // server is responsive
+    public static void stopSyncServer() throws Exception {
         Request request = new Request.Builder()
-                .url(Constants.AUTH_SERVER_URL)
+                .url(STOP_SERVER)
                 .build();
 
-        while (retryTimes != 0) {
-            Response response = null;
+        Response response;
+        int attempts = 3;
```

Comment:
why do you need retries on stop but not start?

Reply:
This was an attempt to fix another issue, but is really the wrong fix. I'll remove the retry again.